### ISSUE TITLE
jupyter_lsp - different location of jupyter-lsp-notebook.json between 2.2.2 and 2.2.3

### ISSUE THREAD

**Issue by frenzymadness** [jupyter-lsp/jupyterlab-lsp#1068]
This is probably more of a question than a bug report. If you install jupyter_lsp 2.2.2, `jupyter-lsp-notebook.json` is in two different locations (in a virtual environment):
```
./lib/python3.12/site-packages/jupyter_lsp/etc/jupyter-lsp-notebook.json
./etc/jupyter/jupyter_notebook_config.d/jupyter-lsp-notebook.json
```
with 2.2.3 it's just in
```
./lib/python3.12/site-packages/jupyter_lsp/etc/jupyter-lsp-notebook.json
```

I maintain this as an RPM package and for version 2.2.2 I moved the config file from its default install location `/usr/etc` to `/etc`.

So my question is: Should I move the config files to /etc or leave them in site-packages?

**Comment by krassowski** [comment]
Yes, in 2.2.3 we partially modernized the setup script/config removing data_files from `setup.py`:

```diff
  setuptools.setup(
-     data_files=[
-         (
-             "etc/jupyter/jupyter_server_config.d",
-             ["jupyter_lsp/etc/jupyter-lsp-jupyter-server.json"],
-         ),
-         (
-             "etc/jupyter/jupyter_notebook_config.d",
-             ["jupyter_lsp/etc/jupyter-lsp-notebook.json"],
-         ),
-     ],
  )
```

and adding it in `setup.cfg` (still not `pyproject.toml`, but some progress):

```diff
+ [options.data_files]
+ etc/jupyter/jupyter_server_config.d = jupyter_lsp/etc/jupyter-lsp-jupyter-server.json
```

Now, you can see that `jupyter-lsp-notebook.json` is no longer there, only `jupyter-lsp-jupyter-server.json`. This is because we no longer support the old `notebook` server and require a modern `jupyter-server`.

The fact that `jupyter_lsp/etc/jupyter-lsp-notebook.json` was not deleted appears to be an omission. I think we should simply remove it in the next patch release.

**Comment by frenzymadness** [comment]
Oh, I see. I thought that both JSONs are the same but you are right that the `jupyter-lsp-jupyter-server.json` is still installed in the same location. Thank you for the quick reply.
We can keep this open as a reminder to remove the old JSON.

**Comment by krassowski** [comment]
jupyter-lsp 2.2.4 should have solved it, please reopen if there is still a problem. Thanks!
